src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -137,8 +125,25 @@ def get_instance_loader(*, class_: type) -> Callable:
     return partial(load_instance, class_=class_)
 
 
+@lru_cache(maxsize=None)
+def get_dict_of_loader(*, class_: type) -> Callable:
+    """
+    Return a load  instance function pre-populate with the class_. @ramon change the docstring
+    """
+
+    def _get_dict_of_loader(alfacase_content: DescriptionDocument, class_: Type[T]) -> T:
+        return {
+            key.data: load_instance(
+                DescriptionDocument(value, alfacase_content.file_path), class_=class_
+            )
+            for key, value in alfacase_content.content.items()
+        }
+
+    return partial(_get_dict_of_loader, class_=class_)
+    
+
 def get_case_description_attribute_loader_dict(
-    class_: Any, explicit_loaders: Optional[Dict[str, Callable]] = None
+    class_: Any, explicit_loaders: Optional[Dict[str, Callable]]=None
```

Comment:
Did this pass by precommit check?
There is a lot of this.

Reply:
hmm, I didn't thought that was why it didnt pass the precommit check, but I'll revert it just to be sure